Apps/Common/src/AspNetConfiguration/ProgramConfiguration.cs

```diff
@@ -54,6 +54,7 @@ public static IWebHostBuilder CreateWebHostBuilder<T>(string[] args)
             .ConfigureAppConfiguration((builderContext, config) =>
             {
                 config.AddEnvironmentVariables(prefix: EnvironmentPrefix);
+                config.AddJsonFile("appsettings.local.json", true, true); //load local settings
```

Comment:
This now enables us to have local configurations which are not in GitHub (store cert passwords or different settings)

Reply:
Wouldn't it be better to use secrets in our dev config?
https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-3.0&tabs=windows